Documentation/network/servicemesh/gateway-api/troubleshooting.rst

```diff
@@ -0,0 +1,166 @@
+    Per Gateway or listener you can always see two status conditions:
+
+      * ``Accepted``: this means that the Gateway configuration was accepted.
+      * ``Programmed``: this means that the Gateway configuration was programmed into Envoy.
+      * ``ResolvedRefs``: this means that all referenced secrets were found and got permission to be used.
+  
+    If any of these are set to false you can get more information by looking at the ``Message`` and ``Reason`` fields.
+
+
+#. Check the HTTPRoute resource
+
+  When our Gateway is functional we can check our routes to see if they are configured correctly.
+  The way to do check the route status is very similar to the Gateway resource. While these instructions are written for HTTPRoute, they also apply to other route types.
+
+  .. code-block:: shell-session
+
+    $ kubectl get httproute -A
+    NAMESPACE                 NAME              HOSTNAMES         AGE
+    website                   homepage          www.example.org   17m
+    webshop                   catalog-service                     17m
+    webshop                   cart-service                        17m
+
+  To get further information we can again use ``kubectl describe httproute <name>``
+
+  .. code-block:: shell-session
+
+    $ kubectl describe httproute <name>
+    Status:
+      Parents:
+        Conditions:
+          Last Transition Time:  2023-06-05T15:11:53Z
+          Message:               Accepted HTTPRoute
+          Observed Generation:   1
+          Reason:                Accepted
+          Status:                True
+          Type:                  Accepted
+          Last Transition Time:  2023-06-05T15:11:53Z
+          Message:               Service reference is valid
+          Observed Generation:   1
+          Reason:                ResolvedRefs
+          Status:                True
+          Type:                  ResolvedRefs
+        Controller Name:         io.cilium/gateway-controller
+        Parent Ref:
+          Group:  gateway.networking.k8s.io
+          Kind:   Gateway
+          Name:   same-namespace
+
+  Here under status we get all the conditions that are relevant for the specific HTTPRoute.
+  These are listed per parent reference to the gateway. If you linked the route to multiple gateways you will see multiple entries here.
+  There are two status conditions that are of interest to investigate:
+
+  * ``Accepted``: This means that the HTTPRoute configuration was correct and accepted.
+  * ``ResolvedRefs``: This means that the referenced services were found and are valid references.
+  
+  If any of these are set to false you can get more information by looking at the ``Message`` and ``Reason`` fields.
+
+Common mistakes
+---------------
+
+
+.. include:: mistakes-warning.rst
+
+* The backend service does not exist. You can check if the backend service was found under the ``conditions`` field when doing a ``kubectl describe httproute <name>``
```

Comment:
formatting, period nit, active voice

What's doing the finding? Is it Kubernetes via kubectl? It's better to be as specific as possible.
```suggestion
* The backend service does not exist. 

    To verify whether the backend service was found, run ``kubectl describe httproute <name>`` and inspect the ``conditions`` field:
```